### PR TITLE
Update episode 04 citation

### PR DESCRIPTION
**Pull request by SarahAlidoost**

### CODE REVIEW

File: _episodes/04-citation.md [hunk 1]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
+
+Citation helps software developers be recognized for their work.
+Additionally, a citation is an integral part of scientific accountability and reproducibility.
 
-## Why make your software citable?
-Citation helps software developers be recognized for their work. Additionally, citation is an integral part of scientific accountability and reproducibility, but accurately citing software is inherently more difficult than citing a paper. To an outsider especially, even seemingly trivial things such as identifying who should be recognized as an author can be difficult. It is therefore convenient when software developers themselves provide the information necessary to enable citation.
+However, citing software is inherently more difficult than citing a paper.
+To an outsider especially, even seemingly trivial things such as identifying
+who should be recognized as an author can be difficult. It is therefore convenient
+when software developers themselves provide the information necessary to enable citation.
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
when software developers themselves provide the information necessary to cite.
```

File: _episodes/04-citation.md [hunk 2]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
-## Getting started
-### Adding a citation file to your code
-By adding a machine readable citation file to your code base, you can define how the software should be cited.
+## A citation file
 
-The [CodeMeta](https://codemeta.github.io/) standard and the [Citation File Format](https://citation-file-format.github.io/) were specifically designed to enable citation. For either one, you write a plain text file with citation metadata, which you then distribute with your software.
+By adding a machine-readable citation file ``CITATION.cff`` to your code base,
+you can define how the software should be cited.
+
+The [CodeMeta](https://codemeta.github.io/) standard and the
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
The [CodeMeta](https://codemeta.github.io/) metadata schema and the
```
CodeMeta is not a standard

File: _episodes/04-citation.md [hunk 3]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
+> > on the [GitHub guides page](https://guides.github.com/activities/citable-code/#repository).
+> {: .solution}
+{: .challenge}
 
-### Create a persistent identifier
-Software is continiously involving, and ideally when someone uses your software, they cite the exact version of the software they use. To facilitate that, you can make a *persistent identifier* (DOI, URN, ARK, etc) for a snapshot of your software, so that the identifier will continue to resolve to exactly that version for the foreseeable future.
+## A persistent identifier
 
-There are several archiving services that help you create such a identifier, either at the push of a button, or automatically, for example each time you make a new release of your software:
+Software is continuously improving, and ideally when someone uses your software,
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
Software is continuously evolving, and ideally when someone uses your software,
```
It was originally evolving and I think by mistake ended up being involving 🤣

File: _episodes/04-citation.md [hunk 4]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
+they cite the exact version of the software they use. To facilitate that,
+you can make a **persistent identifier** (DOI, URN, ARK, etc) for a snapshot of
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
you can make a **persistent identifier**
([Digital Object Identifier or DOI](https://en.wikipedia.org/wiki/Digital_object_identifier),
[Uniform Resource Name or URN](https://en.wikipedia.org/wiki/Uniform_Resource_Name),
[Archival Resource Key or ARK](https://en.wikipedia.org/wiki/Archival_Resource_Key),
etc) for a snapshot of
```

File: _episodes/04-citation.md [hunk 5]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
+your software, so that the identifier will continue to resolve to exactly
+that version for the foreseeable future.
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
that version in the future.
```

File: _episodes/04-citation.md [hunk 6]
```diff
@@ -4,43 +4,91 @@ teaching: 0
...
 
-- [Zenodo](https://zenodo.org/)
-- [FigShare](https://figshare.com/)
-- [Software Heritage Archive](https://softwareheritage.org/)
+> ## Archiving services
+>
+> There are several archiving services that help you create such an identifier,
+> either using the push of a button, or automatically, for example each time
```
Comment (mkuzak) [inline _episodes/04-citation.md]:
```suggestion
> either semi-automatically, or in a fully automated manner, for example, each time
```

**Review by mkuzak — CHANGES_REQUESTED** [review]

**Comment by mkuzak** [comment]
Hi @SarahAlidoost thanks again, great progress. There are a few small things to address before we can merge it.

**Comment by SarahAlidoost** [comment]
> Hi @SarahAlidoost thanks again, great progress. There are a few small things to address before we can merge it.

Thank you @mkuzak for your helpful suggestions. All of them are committed.

**Pull request merged**